server/db/index.js

```diff
@@ -0,0 +1,13 @@
+const mongoose = require("mongoose")
+
+const mongoString = "mongodb+srv://m001-student:m001-mongodb-basics@cluster0.dvh8w.mongodb.net/myFirstDatabase?retryWrites=true&w=majority"
```

Comment:
might not be safe to leave this in the public. Everyone could edit it

Reply:
resolved